test/scripts/e2e_subs/e2e-app-simulate.sh

```diff
@@ -22,7 +22,7 @@ CONST_FALSE="false"
 
 # First, try to send an extremely large "transaction" in the request body.
 # This should fail with a 413 error.
-truncate -s 11MB "${TEMPDIR}/toolarge.tx"
+dd if=/dev/zero of=${TEMPDIR}/toolarge.tx bs=1M count=11
```

Comment:
bs=11M count-1?

Reply:
Changed in [8531284](https://github.com/algorand/go-algorand/pull/5256/commits/85312849f5a7235c74f0f0226628497ecc6118dc)